### PR TITLE
Mylin/fix 435 2

### PR DESCRIPTION
**Pull request by acdo2002**
Close the issue #435

### CODE REVIEW

**Comment by acdo2002** [comment]
Hi @kswang1029 ,

I see, many thanks for the example. 
So I should change the codes to fit the following requirements:
(1) KEEP the cursor info bar "always be visible".
(2) Cursor INSIDE the image, the cursor info bar shows 5 items: WCS, Image, Frequency, VALUE & velocity.
Otherwise (cursor outside the image), the cursor info bar only shows 4 items: WCS, Image, Frequency & velocity.

Please let me know whether my understand is correct.

**Comment by kswang1029** [comment]
@acdo2002 yes

**Comment by veggiesaurus** [comment]
> Hi @kswang1029 ,
> 
> I see, many thanks for the example.
> So I should change the codes to fit the following requirements:
> (1) KEEP the cursor info bar "always be visible".
> (2) Cursor INSIDE the image, the cursor info bar shows 5 items: WCS, Image, Frequency, VALUE & velocity.
> Otherwise (cursor outside the image), the cursor info bar only shows 4 items: WCS, Image, Frequency & velocity.
> 
> Please let me know whether my understand is correct.

@acdo2002 currently, if the `activeFrame`'s  `cursorValue` is undefined, the cursor overlay leaves out the `cursor: <cursorVale>` part of the cursor info bar. I think therefore the easiest way to handle this issue is to simply set the `cursorValue` to undefined when the user moves the cursor out of the image

File: src/stores/FrameStore.ts [hunk 1]
```diff
@@ -244,9 +247,19 @@ export class FrameStore {
             }
         }
 
+        // Is cursorPos inside the image?
+        let isInsideImg = false;
+        if (cursorPosImageSpace.x + ORIGIN_POINT_OFFSET < this.frameInfo.fileInfoExtended.width && 
+            cursorPosImageSpace.x + ORIGIN_POINT_OFFSET > 0 &&
```
Comment (veggiesaurus) [inline src/stores/FrameStore.ts]: you can just use `Math.round(cursorPosImageSpace.x)` can't you?

**Pull request closed without merging**